catalyst-gateway/bin/src/settings/cassandra_db.rs

```diff
@@ -91,6 +103,15 @@ impl EnvVars {
                 MIN_BATCH_SIZE,
                 MAX_BATCH_SIZE,
             ),
+            region_1: StringEnvVar::new(
```

Comment:
AWS config should not be the defaults.  If these are needed for AWS, they should default to not set.
SRE should then set them in production or development as required.

Reply:
addressed with options.